modules/oe_webtools_laco_widget/oe_webtools_laco_widget.module

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal hooks.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\oe_webtools_laco_widget\Event\LacoWidgetConfig;
+
+/**
+ * Implements hook_page_attachments().
+ */
+function oe_webtools_laco_widget_page_attachments(array &$attachments) {
+  /** @var \Symfony\Component\EventDispatcher\EventDispatcherInterface $dispatcher */
+  $dispatcher = \Drupal::service('event_dispatcher');
+
+  $include = ['#page-wrapper'];
+  $exclude = ['.nolaco', '.more-link', '.pager'];
+  $event = new LacoWidgetConfig($include, $exclude);
+  /** @var \Drupal\oe_webtools_laco_widget\Event\LacoWidgetConfig $event */
+  $event = $dispatcher->dispatch(LacoWidgetConfig::LACO_WIDGET_CONFIG_EVENT, $event);
```

Comment:
I don't understand why we use an event to configure this functionality. Doesn't it make more sense to simply use drupal configuration? That way the default configuration can be exported like all the rest and can be easily changed through a form for example.

Reply:
@imanoleguskiza We are using an event so that we can allow implementing sites to alter the classes. But this is not something for editors for example, only devs should be allowed to change this, which means anyway code. And it means also unnecessary form for configuring, config export and stuff.

Reply:
I'm not so sure this will be the work of devs to configure. I'm quite certain webmasters and people from CEM will need to make adjustements to this quite often (at least I believe this was the case on NE sites). Now it may be the case that we are moving away from this approach but I couldn't see this reflected in the tickets so far.

Reply:
@imanoleguskiza Checked on how it's used. Atm it's a simple bean block dump if the json and placed in the footer. So no configurability for now.

Reply:
Yes it's a bean block but the contents of the json are defined by whoever creates the bean, so whoever has the appropriate permission can define what is included and what is excluded. I believe that we are losing functionality here but the base is there so I'll approve for now and follow up with discussion later.